server.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const app = express();
+const port = process.env.PORT || 3030;
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+const Todo = require('./api/models/todoListModels')
+const cors = require('cors');
+
+const corsOptions = {
+  "origin": "*",
+  "methods": "GET,HEAD,PUT,PATCH,POST,DELETE",
+  "preflightContinue": false,
+  "optionsSuccessStatus": 204
+};
+
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://localhost/Tododb', { useMongoClient: true });
+
+app.use(bodyParser.urlencoded({extended: true}));
```

Comment:
we shouldn't need this; I think we should enforce json bodies everywhere for POST/PUT/DELETE/etc. requests. no need for urlencoded bodies.